### PR TITLE
Update the Packs overview

### PR DESCRIPTION
**Pull request by sudkul87**
## Describe the Change

This PR updates the Packs overview on the index page in the docs. 

## Review Changes

💻 [Add Preview URL](https://deploy-preview-1187--docs-spectrocloud.netlify.app/registries-and-packs)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1187#" title="Author self-approved">sudkul87</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/09-registries-and-packs.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs.md:15]:
```suggestion
A **Cluster Profile** is made up of preconfigured layers, each of which is called a pack.  In other words, **Packs** are the building block of a cluster profile to create layers such as operating systems, Kubernetes, network, storage, and add-ons. Packs can be broadly categorized into two types:
```
I like the change but I wonder if we should use a simpler word.

File: content/docs/09-registries-and-packs.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs.md:27]:
```suggestion
Palette provides a rich collection of out-of-the-box packs for various integrations and also offers extensibility through custom-built packs. Each Pack is a collection of files such as manifests, helm charts, Ansible roles, configuration files, etc. Ansible roles, if provided, are used to customize cluster virtual machine images, whereas Kubernetes manifests and Helm charts are applied to the Kubernetes clusters after deployment. The following is a typical structure of a pack:
```
Spelling out what VM is for the reader.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Great first PR @sudkul87. I left some minor suggestions but everything else looks great.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1187#pullrequestreview-1343920769" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1187#" title="Author self-approved">sudkul87</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
